### DISCUSSION TITLE
Open source semver.npmjs.com

### DISCUSSION THREAD

**Discussion by Stargator** [npm/feedback#531] [General]
Would it be possible to open source the web components of semver.npmjs.com? Or does anyone know of an existing open source repo that does a similar job?

We have a closed-source Nexus repo for out internal projects and I think this could be useful for newcomers on our team to practice their semantic versioning understanding.

**Comment by ljharb** [comment]
https://npmjs.com/semver may be helpful.

**Comment by MylesBorins** [comment]
@Stargator the source is not currently OSS but it does feel like something we could look into open sourcing (although. the code is a bit old 😅)

Let me talk to the team and see what we can do

**Comment by Stargator** [comment]
@MylesBorins Thanks. And thank you @ljharb that may be a good library to use if my team wants to put into the effort to build something for newcomers to try out.
